### PR TITLE
test: switch language

### LINKED ISSUE #1290: Switch Language doesn't work 🤔
# Bug report

## Describe the bug

The Language Switcher[1] doesn't work as expected 🤔 

cc @guoqqqi Please take a look

[1] http://139.217.190.60/consumer/list

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
resolve #1290 
___
### New feature or improvement
- Describe the details and related test reports.
add switch language test

### CODE REVIEW

File: web/cypress/integration/lang/switch-language.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Switch language', () => {
+  const timeout = 1000;
+  const domSelectors = {
```
Comment (juzhiyuan) [inline web/cypress/integration/lang/switch-language.spec.js:21]: You have `domSelectors` and `domSelector` in different PRs, we use `domSelector` here.

File: web/cypress/integration/lang/switch-language.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    switch: '.ant-space-align-center',
```
Comment (juzhiyuan) [inline web/cypress/integration/lang/switch-language.spec.js:22]: `switcher`

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

File: web/cypress/integration/lang/switch-language.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    cy.contains('简体中文').click({
+      timeout,
+    });
+    cy.contains('服务').click();
+
+    cy.get(domSelector.switcher).click('right');
+    cy.contains('English').click({
+      timeout,
+    });
+    cy.contains('Create');
```
Comment (liuxiran) [inline web/cypress/integration/lang/switch-language.spec.js:42]: does it need `.should('exist')` here?

Comment (juzhiyuan) [reply web/cypress/integration/lang/switch-language.spec.js:42]: `.should("contain", "TEXT")`?

Comment (juzhiyuan) [reply web/cypress/integration/lang/switch-language.spec.js:42]: @LiteSun Please take a look at here.

File: web/cypress/integration/route/search-route.spec.js [hunk 4]
```diff
@@ -89,8 +89,8 @@ context('Create and Search Route', () => {
     cy.contains('Route').click();
 
     // search one label
-    cy.get('[title=Labels]').click();
     cy.wait(500);
+    cy.get('[title=Labels]').click();
```
Comment (juzhiyuan) [inline web/cypress/integration/route/search-route.spec.js:93]: `domeSelector`

File: web/cypress/integration/route/search-route.spec.js [hunk 5]
```diff
@@ -89,8 +89,8 @@ context('Create and Search Route', () => {
     cy.contains('Route').click();
 
     // search one label
-    cy.get('[title=Labels]').click();
     cy.wait(500);
```
Comment (juzhiyuan) [inline web/cypress/integration/route/search-route.spec.js:92]: Do we really need this wait?

File: web/cypress/integration/route/search-route.spec.js (hunk 4 above)
Comment (guoqqqi) [reply web/cypress/integration/route/search-route.spec.js:92]: This is a test under route, we will change these specifications in the new PR.

File: web/cypress/integration/lang/switch-language.spec.js (hunk 3 above)
Comment (guoqqqi) [reply web/cypress/integration/lang/switch-language.spec.js:42]: You cannot use .should() here, as shown in the figure: 
image

File: web/cypress/integration/route/search-route.spec.js (hunk 5 above)
Comment (guoqqqi) [reply web/cypress/integration/route/search-route.spec.js:92]: yes, we need

Comment (juzhiyuan) [reply web/cypress/integration/route/search-route.spec.js:92]: Why we need?

File: web/cypress/integration/route/search-route.spec.js (hunk 4 above)
Comment (juzhiyuan) [reply web/cypress/integration/route/search-route.spec.js:93]: not conflicted to add this to `domSelector`

**Comment by juzhiyuan** [comment]
image

Those commits are not related to Language Switcher, revert them.

**Pull request merged**